platform_versioned_docs/version-23.3.0/enterprise/prerequisites/on-prem.mdx

```diff
@@ -5,28 +5,30 @@ date: "12 Apr 2023"
 tags: [on-prem, prerequisites, configuration]
 ---
 
-This page describes the prerequisites for deploying Seqera Platform Enterprise to your on-premises infrastructure.
+This page details the prerequisites for deploying Seqera Platform Enterprise to your on-premises infrastructure.
 
 See [Docker Compose](../docker-compose.mdx) for instructions to deploy via Docker Compose.
 
 See [Kubernetes](../kubernetes.mdx) for instructions to deploy via Kubernetes.
 
 ## Prerequisites
 
-Ensure that you satisfy the requirements for your installation. Refer to this list of prerequisites:
+You must satisfy the requirements for your installation:
 
 - **SMTP server**: An SMTP server is required to send emails from your Seqera instance.
 
   If you don't have your own mail server, you can use an external service from a cloud provider. Visit the provider's corresponding **Prerequisites** page for more information and consult your IT team to select the most suitable solution for your organization.
 
-- **MySQL database**: An external database (i.e., external to your Docker Compose or Kubernetes deployment) is highly recommended for production deployments. If you don't have your own database service, you can use an external service from a cloud provider. Visit the provider's corresponding **Prerequisites** page for more information and consult your IT team to select the most suitable solution for your organization.
+- **MySQL database**: An database external to your Docker Compose or Kubernetes environment is highly recommended for production deployments. 
```

Comment:
Is it _an_ or _a_ database?

Reply:
Lol. Thanks. Feel free to suggest fixes or just commit directly in such silly typo cases :)